.github/workflows/pr-checks.yml

```diff
@@ -26,17 +26,26 @@ jobs:
           find . -type f -name '*.sh' | xargs shellcheck --external-sources
 
   digest-validation:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@v1
     - name: Run script which checks container image digest
       run: |
         sudo pip install yq
         /bin/bash .ci/devfile-images-check.sh
 
-  build:
+  build-dev-containers:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v1
+
+      - name: Build dev container images
+        run: ./arbitrary-users-patch/build_images.sh --rm
```

Comment:
if you remove them but don't publish them, how will the subsequent step be able to resolve them as base image for the happy path image build?

Reply:
If we publish them then every PR run will overwrite the `nightly` tag for all the dev containers. Also in this case we aren't building the HP image, only the dev containers and the main devfile registry image. It's different than the release job/script.

Reply:
yep, sounds good.